std/random.d

```diff
@@ -3023,17 +3032,17 @@ private struct RandomCoverChoices
     {
         RandomCoverChoices c = RandomCoverChoices(length);
         assert(c.hasPackedBits == (length <= size_t.sizeof * 8));
-        c[0] = true;
-        c[2] = true;
-        assert(c[0]);
-        assert(!c[1]);
-        assert(c[2]);
-        c[0] = false;
-        c[1] = true;
-        c[2] = false;
-        assert(!c[0]);
-        assert(c[1]);
-        assert(!c[2]);
```

Comment:
It is necessary to change this unittest?

Reply:
Removed change.